File: c++/easy/mergestringsalternately.cpp
#include <iostream>

class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        string str = "";
        int count = 0;

        while(count < word1.length() || count < word2.length()) {
            if(count < word1.length()) {
                str += word1[count];
            } 
            
            if(count < word2.length()) {
                str += word2[count];
            }

            count += 1;
        }

        return str;
    }
};

File: python/easy/mergestringsalternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        str1 = ""
        count = 0

        while count < len(word1) or count < len(word2):
            if count < len(word1):
                str1 += word1[count]

            if count < len(word2):
                str1 += word2[count]

            count += 1

        return str1
